packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.scss

```diff
@@ -4,29 +4,49 @@
 .product-payment {
   background: $color-white;
   border: 1px solid #e7e7e7;
+  border-bottom: 0;
+  flex: 1;
   margin: 30px 0;
   max-width: 600px;
   min-width: 375px;
-  padding: 28px 16px 60px;
+  padding: 16px 16px 60px;
 
-  @include min-width('tablet') {
-    padding: 28px 48px 48px;
-  }
-
-  @include min-width('desktop') {
-    margin: 30px 24px;
-  }
+  .billing-title {
+    font-size: 15px;
+    font-weight: 600;
+    margin-bottom: 20px;
```

Comment:
I think this actually needs a color, as does "Set up your subscription" and "total price." They look black and should be dark grey.